lib/ngx/ocsp.lua

```diff
@@ -21,6 +22,8 @@ local FFI_BUSY = base.FFI_BUSY
 
 
 ffi.cdef[[
+typedef long time_t;
```

Comment:
This may interfere with other FFI-based Lua modules. Let's not try to define common types like `time_t` ourselves. Just use built-in type names like `long` directly.